pallets/order-book/Cargo.toml

```diff
@@ -67,3 +67,5 @@ runtime-benchmarks = [
 try-runtime = ["frame-support/try-runtime"]
 
 ready-to-test = ["framenode-chain-spec/ready-to-test", "framenode-runtime/ready-to-test", "common/ready-to-test"]
+
+private-net = []
```

Comment:
```suggestion
private-net = [
    "framenode-chain-spec/private-net",
    "framenode-runtime/private-net",
]
```

Reply:
Yeah, it makes sense; I've added them previously, but this compilation error was happening with these flags in chainspec+runtime: https://jenkins.soramitsu.co.jp/blue/organizations/jenkins/sora2%2Fsubstrate/detail/PR-816/66/pipeline

need to debug reason for it and fix, agree